src/data-table/index.js

```diff
@@ -63,10 +65,10 @@ class DataTable extends Component {
   }
 
   downloadReport = () => {
-    const { data, downloadName } = this.props
+    const { data, downloadName, downloadPicked } = this.props
 
-    const headers = this.pickedColumns().map(c => c.name)
-    const valueKeys = this.pickedColumns().map(c => c.dataKey)
+    const headers = downloadPicked ? this.pickedColumns().map(c => c.name) : this.columns().map(c => c.name)
+    const valueKeys = downloadPicked ? this.pickedColumns().map(c => c.dataKey) : this.columns().map(c => c.name)
```

Comment:
This should be `this.columns().map(c => c.dataKey)` for the else case.

To eliminate such problem, we can make one reusable `const columns` to make code more concise:

```js
const columns = downloadPicked ? this.pickedColumns() : this.columns()
const headers = columns.map(c => c.name)
const valueKyes = columns.map(c => c.dataKey)
```